tests/k8s_otel/conftest.py

```diff
@@ -0,0 +1,52 @@
+import pytest
+import os
+from pathlib import Path
+from pytest_kind import KindCluster, cluster
+
+from jina.logging.logger import JinaLogger
+from .kind_wrapper import KindClusterWrapperV2
+
+# The default version broke cni at some point. That's why we need to specify the version here.
+# This can and probably should be put in env variable actually.
+cluster.KIND_VERSION = 'v0.11.1'
```

Comment:
You can probably run the subprocess command to get the KIND version.

Reply:
Oh this isn't to read the kind version, it is to get it to download the right kind version. I took it from the other k8s tests.

I saw in the source code for pytest_kind that it actually reads it from env variable so we can set it in the CI instead of in the code. I think that might be better.